src/styles/components/_pager.scss

```diff
@@ -4,10 +4,16 @@ $dso-pager-height: $u6;
 
 .pager {
   li {
+    > a,
+    > span {
+      border-radius: $dso-button-border-radius;
```

Comment:
Dit komt dus overeen met secundaire knop (dus: `.btn-default`)? Is het mogelijk een mixin aan te (gaan) roepen?

Reply:
Klopt, en na wat onderzoek bleek het inderdaad te gaan om de hele "basis" van de default knop. Daar is nu een mixin van gemaakt.